resources.js

```diff
@@ -0,0 +1,66 @@
+/*Location to retrieve resources for each waifu theme*/
```

Comment:
Does this file need to be outside of the firefox themes?

Reply:
Outside of the waifus folder? No. This file can be placed anywhere, for it is a background script. So long as the background key in the manifest file is pointing to this file.

Reply:
Not sure what you mean with this comment. Do you mean the resources.js need to be outside of the firefox directory?

Reply:
Sorry about the confusion. This has to do with both `resources.js` and `waifus/waifu.js`.

I was wondering why it was outside of the folder. The PR has 2 of these, one inside of the `fireFoxThemes` directory and one here.

![image](https://user-images.githubusercontent.com/15972415/102688305-9d60f900-41bb-11eb-82ec-6800ec1add9d.png)

> The above is a diff between your master and my master.


I think it may be an artifact of you merging your standalone [firefox](https://github.com/ZimCodes/doki-theme-firefox/tree/firefox) branch into master.

Reply:
Yes, this is exactly what occurred when i merged my firefox branch with my master. I'll clean this up.